pipe/runtime/src/scheduler.rs

```diff
@@ -177,6 +213,7 @@ where
             None => ScheduleResult::New,
         };
         self.pipe_configs.insert(id, config);
+        self.unschedule(id).await;
```

Comment:
Why does this unschedule and then reschedule here?

Reply:
unschedule makes sure that current pipe handle (if any) is definitely aborted, i.e. if pipe in restart state - timer should never fire if we re-added pipe

Reply:
@wwkeyboard let me know if this call is still a concern